<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LongQtMainWindow</class>
 <widget class="QMainWindow" name="LongQtMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>LongQt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QPushButton" name="aboutButton">
      <property name="text">
       <string>About</string>
      </property>
      <property name="icon">
       <iconset theme="&quot;help-about&quot;"/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="nextButton">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListWidget" name="menuList"/>
      <widget class="QStackedWidget" name="menuStack">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
